sql/core/src/test/java/test/org/apache/spark/sql/JavaDatasetSuite.java

```diff
@@ -34,6 +34,8 @@
 import org.apache.spark.sql.catalyst.encoders.Encoder$;
```

Comment:
We should do something about this.  I think that if we rename the object to `Encoders` then Scala will build static forwarders and we can just call `Encoders.STRING()`